services/restaurants/restaurant_comparer.py

```diff
@@ -0,0 +1,143 @@
+from typing import Dict
+from models.restaurants_model import Restaurant
+from constants import API_TOKEN
+from yelp_fusion_api.yelp_fusion import YelpFusion
+import nltk
+from nltk.corpus import wordnet
+import re
+
+class RestaurantComparer:
+
+    def __init__(self):
+        self.curr_resturant = None
+
+    def compare_restaurants(self, rest1, rest2):
+        # takes in restaurant 1 id & restaurant 2 id
+        headers = {
+            "Authorization": f"Bearer {API_TOKEN}"
+        }
+        # finds restaurant info
+        restInfo1 = YelpFusion().find_restaurant_by_id(id=rest1, headers=headers)
+        restInfo2 = YelpFusion().find_restaurant_by_id(id=rest2, headers=headers)
```

Comment:
handle error here if requests return 401 errors because restuarant id cant be found or doesn't exist

Reply:
fixed